core/src/test/java/org/apache/iceberg/TestPartitioning.java

```diff
@@ -140,7 +140,7 @@ public void testPartitionTypeWithAddingBackSamePartitionFieldInV1Table() {
         .addField("data")
         .commit();
 
-    // in v1, we use void transforms instead of dropping partition fields
+    // in v1, we use void transforms instead of dropping partition fields.
```

Comment:
Please remove this change since it is non-functional. Also, we don't require punctuation to end comments.

Reply:
reverted